habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/PythonVersionHelper.java

```diff
@@ -41,50 +42,81 @@ public PythonVersionHelper(File workingDirectory, String desiredPythonVersion) {
      * @return
      */
     public String getCurrentPythonVersion() throws MojoExecutionException {
-	String version = execute(Collections.singletonList("--version"));
-	return version.replaceAll(extractVersionRegex, "");
+        String version = quietlyExecute(Collections.singletonList("--version"));
+        return version.replaceAll(EXTRACT_VERSION_REGEX, "");
     }
 
     /**
      * Executes a python command with the given arguments, logs the executed
-     * command, and returns the resultant process output as a string.
+     * at DEBUG level, and returns the resultant process output as a string.
```

Comment:
I: creature comfort so we just have less output since this now runs every time we get the pyenv version